pkgs/applications/virtualization/rkt/default.nix

```diff
@@ -2,14 +2,14 @@
 , fetchurl, fetchFromGitHub }:
 
 let
-  coreosImageRelease = "794.1.0";
-  coreosImageSystemdVersion = "222";
+  coreosImageRelease = "835.9.0";
```

Comment:
Got it, I'll make a PR to switch to that. I had assumed that it just used the latest stable.